dbfit-java/build.gradle

```diff
@@ -134,16 +204,39 @@ subprojects {
             }
     }
 
-    // to be overriden when needed
-    ext.connectionPropertiesFiles = []
-
     ext.copyConnectionProperties = {
         destination ->
             project.connectionPropertiesFiles.each { file ->
                 copyProperties new File(project.projectDir, file), destination
             }
     }
 
+    ext.execSqlScriptTestDatabase = {
+        scriptFile ->
```

Comment:
I'm not sure is there anything well known. I think for closures it may be fine to do something like:
``` groovy
ext.execSqlScriptTestDatabase = { scriptFile ->
    ...
}
```

If it's longer list of parameters - something like what we've done so far seems OK to me.